### ISSUE TITLE
Use the WGAN?

### ISSUE THREAD

**Issue by jzijin** [leftthomas/SRGAN#61]
Did you use WGAN?

` netG.zero_grad()`
            `g_loss = generator_criterion(fake_out, fake_img, real_img)`
           ` g_loss.backward()         `   
            `fake_img = netG(z)`
            `fake_out = netD(fake_img).mean()`
I don't understand why caculate the fake_img and fake_out after the g_loss have backward,

`real_out = netD(real_img).mean()`
 `fake_out = netD(fake_img).mean()`
 `d_loss = 1 - real_out + fake_out`
What is the "1" means in the d_loss = 1 - real_out + fake_out?
Is it the gradient penalty? If it is the gradient penalty, Why fix it to 1?

**Comment by leftthomas** [comment]
@jzijin 
Duplicate of #46
